systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -106,40 +106,39 @@
 class KafkaST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(KafkaST.class);
-
+    private static final String TEMPLATE_PATH = TestUtils.USER_PATH + "/../examples/templates/cluster-operator";
     public static final String NAMESPACE = "kafka-cluster-test";
+    private static final String OPENSHIFT_CLUSTER_NAME = "openshift-my-cluster";
```

Comment:
why it's global constant if it's used in one test?

Reply:
I used this for the deletion, if you don't agree, I can remove it.